builder/Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV \
   PROMPT_DIRTRIM=3
 
 # developer tools
+ENV CHANGE_HERE_TO_FORCE_REBUILD_FROM_STEP1=1

```

Comment:
To me, your change implies that the mere existence of a value means it would rebuild from that step.  

The value, however, is not a boolean.  The symbol could take any value (it is ignored).  If the value is _changed_, the rebuild of the image starts from that point.

I'll add comments, shorten the names, and set values that are obviously not booleans.

Reply:
see 8c264da